### ISSUE TITLE
Cleanup display of HID contacts on Events

### ISSUE THREAD

**Issue by emmajane** [humanitarianresponse/site#269]
Now that Events (and Assessments) can display contact details for HID, we may need to do some cleanup on the display.

# Tasks

- Remove padding from the pane.
- Update style so that the person's name is not coloured as a link (this is because of H4 tags -- but this is a sub-heading of "Contact(s)" which is just bold so the style is confusing)

![2016-01-04 at 19 01 1](https://cloud.githubusercontent.com/assets/1299320/12097914/770898aa-b316-11e5-970a-2fdc5a5e7377.png)

# Additional Details

Generally, events will show HR.info details OR HID details, not both, so this is not urgent to cleanup.

Used the following meeting to add both contacts:
https://www.humanitarianresponse.info/en/operations/nepal/event/kathmandu-inter-cluster-coordination-meeting-0

![2016-01-04 at 19 05](https://cloud.githubusercontent.com/assets/1299320/12097899/6257a9be-b316-11e5-9da6-3109e982382f.png)

This is a follow-up issue from #195

**Comment by pcambra** [comment]
After #265 there will be no hr contacts anymore.

Styling of the contact display is a matter of opinion at this point, we have no design for this page or frontend person in the team.

**Comment by humphreymburu** [comment]
@emmajane Should continue removing padding for the HID contacts display ?

**Comment by emmajane** [comment]
@humphreymburu Yes, please proceed with the ticket as it was originally described. Thanks for checking.

**Comment by emmajane** [comment]
@pcambra There is currently a style implemented on the page, which we will continue to follow when the new HID contacts have been implemented.
